### DISCUSSION TITLE
Podcast interview

### DISCUSSION THREAD

**Discussion by ultimike** [VSCodium/vscodium#1539] [Q&A]
Hello,

I'd like to interview one of the maintainers of VSCodium for a podcast that I host.

I have a Drupal-centric podcast - many folks in the Drupal community us VSCode for PHP development, and I'd like to talk to someone about differences, advantages, and other related topics of VSCodium.

I recently emailed @daiyam directly, but haven't heard back :(

-mike

**Comment by daiyam** [comment]
@ultimike Sorry, I'm not always up to date with my secondaries email addresses...

The differences are:
- disabled telemetry
- wider support of OS/Installer
- use of OpenVSX

The advantages:
- disabled telemetry
- still supporting Win7, just added support to ppc64le (for remote hosts)

The disadvantages:
- lack of support for some extensions, primarily the proprietary ones from Microsoft (https://github.com/VSCodium/vscodium/wiki/Extensions-Compatibility)
- lack of integration with GitHub

**Reply by ultimike** [reply]
No problem about the email stuff.

Thanks for the info - would you like to discuss all of this on the podcast?

-mike
